### DISCUSSION TITLE
Two template error messages can be improved

### DISCUSSION THREAD

**Discussion by markfirmware** [alpinejs/alpine#983] [3. Feature ideas]
https://codepen.io/markfirmware/pen/eYdrzoK

1. multiple element roots should read as zero when there are none
2. multiple element roots that are just text could have the message remind the programmer that text looks singular but is not

**Comment by KevinBatdorf** [comment]
Your HTML is a bit malformed and you have `x-data` on template tags, but sure there's room for improvement on the error reporting in general. It might be a good PR if you can both explain and demonstrate it more clearly and with good examples

**Reply by markfirmware** [reply]
Good enough. As always thanks for the detailed service!
